spec/system/reporting/processed_report_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe 'Processed Report' do
```

Comment:
Are you planning on adding a spec that checks that we return the right values? Or does that already exist from a previous PR?

Reply:
I think we'll check these on the the report unit specs, but assume well not be using the search for these values so thought I'd wait for that refactor.